web-components/src/components/footer/footer-new.tsx

```diff
@@ -123,21 +123,21 @@ const Footer: React.FC<{
               linkComponent={LinkComponent}
             />
           </Grid>
-          <Grid item xs={12} sm={6} md={3} lg={2} sx={{ mt: 5 }}>
+          <Grid item xs={12} sm={6} md={3} lg={2} sx={{ mt: { sm: 0, md: 5 } }}>
```

Comment:
Minor nit: I like to extract these to DRY things up a bit. Something like:
```tsx
const sxs = {
  footerItem: { mt: { sm: 0, md: 5 } } as SxProps
}
```
Then calling like
```tsx
sx={sxs.footerItem}
```